js-core/homeworks/phoneApp/src/main.js

```diff
@@ -0,0 +1,111 @@
+function PhoneApp() {
+    this.storage = [];
+}
+
+PhoneApp.prototype.checkNumber = function(numberToCheck) {
+    const toStringThePredictNumber = numberToCheck.toString();
+    const toArrayTheString = toStringThePredictNumber.split('');
+    let flag;
+    
+    toArrayTheString.forEach(elem => {
```

Comment:
it's better to use another JS method, forEach seems bad here, what about *.some/.filter* or any other ?